### PR TITLE
Correctly deal with the compiled file

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: bootstrap/autoload.php [hunk 1]
```diff
@@ -27,9 +27,11 @@
 |
 */
 
-$compiledPath = __DIR__.'/../vendor/compiled.php';
-
-if (file_exists($compiledPath))
+if (file_exists($compiledPath = __DIR__.'/../vendor/compiled.php';))
```
Comment (taiyaei) [inline bootstrap/autoload.php]: This shouldn't compile, extra `;`

File: bootstrap/autoload.php [hunk 2]
```diff
@@ -27,9 +27,11 @@
...
+{
+	require $compiledPath;
+}
+elseif (file_exists($compiledPath = __DIR__.'/../storage/framework/compiled.php';))
```
Comment (taiyaei) [inline bootstrap/autoload.php]: same

File: bootstrap/autoload.php (hunk 1 above)
Comment (GrahamCampbell) [inline bootstrap/autoload.php]: thanks :)

**Comment by taiyaei** [comment]
:+1: for this :)

**Pull request merged**